src/RA_StringUtils.h

```diff
@@ -11,8 +11,8 @@ _NODISCARD std::string Narrow(_In_z_ const wchar_t* wstr);
 _NODISCARD std::wstring Widen(_In_ const std::string& str);
 _NODISCARD std::wstring Widen(_In_z_ const char* str);
 
-[[gsl::suppress(f.6), nodiscard]] std::string Narrow(std::wstring&& wstr) noexcept;
-[[gsl::suppress(f.6), nodiscard]] std::wstring Widen(std::string&& str) noexcept;
+[[ gsl::suppress(f .6), nodiscard ]] std::string Narrow(std::wstring&& wstr) noexcept;
+[[ gsl::suppress(f .6), nodiscard ]] std::wstring Widen(std::string&& str) noexcept;
```

Comment:
I'm assuming the auto-formatter changed these. Is the space between "f" and ".6" going to cause problems?

Reply:
I don't think it does, I ran the analyzer a few times with the existing rules and they didn't pop up.